fineract-provider/src/main/resources/db/changelog/tenant/changelog-tenant.xml

```diff
@@ -165,4 +165,5 @@
     <include file="parts/0143_add_accrual_activity_posting_job.xml" relativeToChangelogFile="true" />
     <include file="parts/0144_transaction_summary_with_asset_owner_report_unc_allocation_fix.xml" relativeToChangelogFile="true" />
     <include file="parts/0145_job_short_name.xml" relativeToChangelogFile="true" />
+    <include file="parts/0147_enable_payment_hub_integration.xml" relativeToChangelogFile="true" />
```

Comment:
Can you rebase this file? Version 146 is missing

Reply:
It is already rebased. version 146 is not the part of file changelog-tenant.xml. It is the part of final-changelog-tenant.xml